### ISSUE TITLE
Remove "Start" Button Requirement For Starting Tournaments

### ISSUE THREAD

**Issue by NikAlexander** [BattlePro/battlepro-issues#74]
Currently, tournaments can only begin if the tournament host clicks the 'start' button located on the tournament page.  This additional step is not needed, if tournament hosts wish to contact teams for check ins they should be able to do so without limitation.  Other sites like battlefy and smash.gg do not require hosts to "start" a tournament, it automatically starts based on the start time submitted for the tournament.

**Enhancement:**
- Remove 'Start' button requirement for starting tournaments
- Allow tournaments to start automatically based on start time submitted during tournament creation

![start](https://user-images.githubusercontent.com/6005869/29802620-8153542c-8c44-11e7-920d-3198f34841b1.JPG)

**Comment by JakeBooher** [comment]
I don't think this is something we can currently do as there is a lot that happens when the tournament host presses that "Start" button that we can't really automate reliably with XenForo's cron system. I've marked this as futurefix as this is something that would require some fairly substantial changes and doesn't seem necessary for the MVP

**Comment by NikAlexander** [comment]
@JakeBooher Could we just switch the "publish" button functionality to the "start" button?  That would do the same thing I think.

**Comment by JakeBooher** [comment]
Unfortunately no, the Publish button only shows up when managing the tournament and makes it so the tournament is publicly visible and locks in all of the changes. The start button finalizes the brackets and does some cleanup before the tournament actually begins. Once a host has published a tournament they will not be able to go back to the manage pages to see the "Publish" button again, only administrators of the site are able to modify a tournament after it's been published
